app/models/user.rb

```diff
@@ -25,4 +25,8 @@ def self.active_merchants
   def self.inactive_merchants
     where(active: false, role: 'merchant').order(:name)
   end
+
+  def self.all_merchants
+    where(role: 1)
```

Comment:
```suggestion
    where(role: "merchant")
```
Let's see if this works -- more readable